### PR TITLE
CLI tool

### PR DESCRIPTION
**Pull request by innabugw**
CLI tool done. Available for review

### CODE REVIEW

File: src/test_cli.ts [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            BusFactor_Latency: busFactorLatency.toFixed(3),
+            ResponsiveMaintainer: responsiveMaintainer.toFixed(2),
+            ResponsiveMaintainer_Latency: responsiveMaintainerLatency.toFixed(3),
+            License: license === 1 ? "Pass" : "Fail",
+            License_Latency: licenseLatency.toFixed(3)
+        };
+
+
+        // Output result to stdout
+        console.log(JSON.stringify(result));
```
Comment (kevastator) [inline src/test_cli.ts:97]: Have you confirmed this will write to stdout?

**Review by kevastator — COMMENTED** [review]
Very good work! I just have a comment about stdout that I want some confirmation on.  Thanks!

File: src/test_cli.ts (hunk 1 above)
Comment (innabugw) [reply src/test_cli.ts:97]: yessir

**Review by kevastator — APPROVED** [review]
If this properly prints to stdout everything looks good

**Pull request merged**